monorail.py

```diff
@@ -1,45 +1,18 @@
-#!/usr/bin/env python3
 from oauth2client.client import GoogleCredentials
```

Comment:
Did you also remove the executable bit of this file (-x)?

Reply:
It wasn't executable to begin with, oddly enough :)